documentation/development/cd.md

```diff
@@ -0,0 +1,15 @@
+# Continuous Delivery
+
+This describes aspects of Continuous Delivery setup specifically for OpenCraft,
```

Comment:
@kewne, I don't think we are doing `Continuous Delivery` with our current setup for Ocim and the cleanup job has nothing to do with it either. Imho, it is a part of `Continuous Integration`